.github/workflows/testing.yml

```diff
@@ -82,7 +82,7 @@ jobs:
         run: |
           set -ex
           cd deploy/codex
-          docker-compose up -d db redis gitlab edm houston acm
+          docker-compose up -d db redis gitlab edm houston acm elasticsearch
```

Comment:
Do you have a ballpark on resource consumption? Probably not much... until we get a big ol' database I imagine.